src/main/java/training/supportbank/Userinput.java

```diff
@@ -8,25 +8,26 @@ public class Userinput {
     private Scanner sc = new Scanner(System.in);
     public boolean running = true;
     public boolean requestUserInput () throws IOException, ParseException {
+        ListMaker lists = new ListMaker();
         CsvReader csv = new CsvReader();
-        csv.test();
+        csv.csvListGenerator(lists.getTransactionsList(),lists.getAllUsers(),"src/main/Transactions2014.csv");
 
 
              //System.in is a standard input stream
             System.out.print("Do you want to (A)List All names of people or (B)List all transactions - Press a or b:");
             //reads string
             String str = sc.nextLine();
-            ArrayList<Transaction> allTransactions =  csv.getTransactionsList();
-            ArrayList<User> allUsers =  csv.getAllUsers();
+            //ArrayList<Transaction> allTransactions =  csv.getTransactionsList();
+            //ArrayList<User> allUsers =  csv.getAllUsers();
             //if user types a in console system to display all names
             if (str.equals("a")) {
 
-                CalcualateBalances.calcBalance(allTransactions, allUsers);
+                CalcualateBalances.calcBalance(lists.getTransactionsList(), lists.getAllUsers());
                 runAgain();
             } else if (str.equals("b")) {
                 System.out.println("Please enter name of User?");
                 String nameLookUp = sc.nextLine();
-                ListAccount.listingPrint(allTransactions, allUsers, nameLookUp);
+                ListAccount.listingPrint(lists.getTransactionsList(), lists.getAllUsers(), nameLookUp);
```

Comment:
This looks well structured and easy to read, good extraction of the code into separate methods